src/System.Private.CoreLib/shared/System/IO/FileStream.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Github won't let me comment on that line but the FileStream constructors shouldn't accept null on the `SafeFileHandle` -- however, we're not checking it for null on the argument validation, we call, `ValidateAndInitFromHandle` which just checks for `handle.IsInvalid`, which if you pass null, causes a NRE. https://github.com/dotnet/coreclr/pull/23703/files#diff-de7b810cbc5a45afcb20e7f2e8cf919dR133

Just got a simple repro now:

```
class Program
    {
        static void Main(string[] args)
        {
            FileStream s = new FileStream(null, FileAccess.ReadWrite);
            Console.WriteLine(s.ToString());
        }
    }
```